Gemfile.lock

```diff
@@ -561,7 +561,9 @@ GEM
     mime-types (3.3.1)
       mime-types-data (~> 3.2015)
     mime-types-data (3.2021.0212)
-    mimemagic (0.3.5)
+    mimemagic (0.3.10)
+      nokogiri (~> 1)
```

Comment:
This is a strange version dependency. Maybe this means that we will have a version 3.11 soon? 🤔

Reply:
They seem to be here to stay:
 - `nokogiri` was added in `0.3.8`
 - `rake` was added in `0.3.9`
 https://github.com/mimemagicrb/mimemagic/blob/master/CHANGELOG.md#039-2021-03-25